spring-cloud-netflix-core/src/main/resources/META-INF/spring.factories

```diff
@@ -15,7 +15,8 @@ org.springframework.cloud.bootstrap.BootstrapConfiguration=\
 org.springframework.cloud.netflix.config.DiscoveryClientConfigServiceBootstrapConfiguration
 
 org.springframework.cloud.client.discovery.EnableDiscoveryClient=\
-org.springframework.cloud.netflix.eureka.EurekaDiscoveryClientConfiguration
+org.springframework.cloud.netflix.eureka.EurekaDiscoveryClientConfiguration,\
+org.springframework.cloud.netflix.eureka.EurekaApplicationEventAutoConfiguration
```

Comment:
:-) I guess I'm surprised that putting extra configuration here works. It wasn't intended to work.